src/Views/Menu.java

```diff
@@ -190,8 +200,8 @@ private String dateInput(boolean start) {
      * promotion list.
      * @return - user input
      */
-    private int intInput() {
-        System.out.print("Enter number of months since last purchase: ");
+    private int intInput(String message) {
+        System.out.print(message);
         Scanner input = new Scanner(System.in);
```

Comment:
can you reuse the scanner object of the class?

Reply:
this one is using the Scanner object of the class, it's just a separate function for prompting the user for a specific int value similar to the Date request function.  I needed one that I could pass a specific prompt to and get an int back.

Reply:
for the new function I just duplicated the one for getting Dates for the most/least popular product option